src/Type/Definition/InputObjectField.php

```diff
@@ -51,8 +60,9 @@ public function getType(): Type
             /**
              * @see it('rejects an Input Object type with incorrectly typed fields')
              */
+
              // @phpstan-ignore-next-line schema validation will catch a Type that is not an InputType
-            $this->type = Schema::resolveType($this->config['type']);
+            return $this->type = Schema::resolveType($this->config['type']);
```

Comment:
maybe the return here is a bit confusing as it's used with assignment and the following line seems to handle it.

Reply:
I started introducing this early return where the code is:

```diff
if (! isset($this->lookup[$key]) {
-   $this->lookup[$key] = $this->produceValue($key);
+   return $this->lookup[$key] = $this->produceValue($key);
}

return $this->lookup[$key];
```

In the previous example, it should obviously improve performance - but even in this case I think it does. Property lookups are not free.